aten/src/ATen/core/function_schema.h

```diff
@@ -56,7 +54,15 @@ struct Argument {
     return alias_info_;
   }
   bool is_inferred_type() const {
-    return is_inferred_type_;
+    bool is_inferred_type = false;
+    if (type_){
```

Comment:
`type_` should never be `nullptr`. If it is, then something has gone wrong. I think we can replace this check with a `TORCH_INTERNAL_ASSERT` to eliminate one level of code nesting.